### ISSUE TITLE
Fixture "loop" not found at v 1.5.0

### ISSUE THREAD

**Issue by zhanymkanov** [yunstanford/pytest-sanic#38]
```python3
@pytest.fixture
def test_cli(loop, test_app, sanic_client):
    return loop.run_until_complete(sanic_client(test_app))
```
Raises `fixture 'loop' not found`

in 1.1.2: `pytest --fixtures` lists all `pytest-sanic`'s builtin fixtures in `fixtures defined from pytest_sanic.plugin`
in 1.5.0:  `pytest --fixtures` lists nothing from pytest-sanic

Reinstallation did not help.

**Comment by zoliszeredi** [comment]
same for me, issue is breaking our builds with 1.5.0

**Comment by rogiervandergeer** [comment]
The `entry_points.txt` is missing from the wheel. This prevents `pytest` from finding and loading the plugin.

**Comment by ashleysommer** [comment]
+1, same problem occurring for me

**Comment by yunstanford** [comment]
Sorry guys, i just was trying to switch to poetry for build system, and missed adding entry_point. I'll do another release soon

**Comment by yunstanford** [comment]
just released 1.6.0
